rskj-core/src/main/java/co/rsk/rpc/modules/txpool/TxPoolModuleImpl.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.rpc.modules.txpool;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import org.ethereum.core.PendingState;
+import org.ethereum.facade.Ethereum;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TxPoolModuleImpl implements TxPoolModule{
+
+    private Ethereum eth;
+    private PendingState pendingState;
+    private ObjectMapper mapper;
+    private final JsonNodeFactory jsonNodeFactory = JsonNodeFactory.instance;
+
+    public TxPoolModuleImpl(Ethereum eth, PendingState pendingState) {
+        this.eth = eth;
+        this.pendingState = pendingState;
+        this.mapper = new ObjectMapper();
+    }
+
+    @Override
+    public String content() {
+//      return "{"pending": {}, queued: {}}";
```

Comment:
Maybe it would be better to add a JavaDoc explaining the return values, instead of this comment.

Reply:
:+1: